dingo/core/utils/plotting.py

```diff
@@ -38,6 +48,8 @@ def plot_corner_multi(
         "levels": [0.5, 0.9],
         "bins": 30,
     }
+    if "truths" in kwargs and kwargs["truths"] is not None and "truth_color" not in kwargs:
```

Comment:
I think it's better to just forward all of the settings related to truth values to `corner.corner`, rather than trying to tweak the appearance. It keeps the Dingo code more streamlined. It's okay to use the corner defaults and not include "Truth" in the legend.